akka-remote/src/main/scala/akka/remote/artery/SystemMessageDelivery.scala

```diff
@@ -30,14 +30,15 @@ import akka.actor.ActorRef
 import akka.dispatch.sysmsg.SystemMessage
 import scala.util.control.NoStackTrace
 
+import akka.annotation.InternalApi
 import akka.event.Logging
 import akka.stream.stage.StageLogging
 import akka.util.OptionVal
 
 /**
  * INTERNAL API
  */
-private[remote] object SystemMessageDelivery {
+@InternalApi private[remote] object SystemMessageDelivery {
```

Comment:
isn't `@InternalApi` superfluous since it is already `private[remote]`?

Reply:
No, because:

- java
- some day we'll finally update MiMa to understand the annotation